### ISSUE TITLE
[C++] Windows build support

### ISSUE THREAD

**Issue by chaokunyang** [apache/fury#798]
## Is your feature request related to a problem? Please describe.
Fury support macos and linux build, but windows is not supported.

Fury use bazel for building c++ code, windows building need some extra work.

**Comment by LiangliangSui** [comment]
Can I try to do some work on this? @chaokunyang

**Comment by chaokunyang** [comment]
> Can I try to do some work on this? @chaokunyang

Of course. Welcome to try this issue. Thanks for the willingness to contribute to Fury.

**Comment by chaokunyang** [comment]
I haven't try bazel on Windows.  If you want to try this issue, you may need to install bazel on Windows first, and fix some compilation issues.
